Sami/Parser/NodeVisitor.php

```diff
@@ -336,4 +343,74 @@ protected function resolveAlias($alias)
         // a class in the current class namespace
         return $class->getNamespace().'\\'.$alias;
     }
+
+    protected function resolveExample(NodeAbstract $node, DocBlockNode $comment) {
+        $example = $comment->getTag('example');
+        if (empty($example)) {
```

Comment:
I try to avoid using `empty` as a lot of things are empty for PHP. I would do `'' === $example` instead